src/styles/card-container.scss

```diff
@@ -0,0 +1,144 @@
+.card-container {
+  color: #060b35;
+  gap: 48px 46px;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+
+  place-content: center;
+
+  padding: 50px 40px;
+}
+
+@media (min-width: 488px) {
+  .card-container {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .card-container {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  .card-container {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.product-card {
+  display: flex;
+  box-sizing: border-box;
+  width: 200px;
+
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  flex-direction: column;
+  padding: 32px 16px 16px;
+
+  transition: all 0.3s;
+}
+
+.product-card:hover {
+  transform: scale(1.2);
```

Comment:
It's not a good idea to use `transform: scale(1.2);` on hover. It can lead to the element overlapping other elements. It's better to change some color properties or add a shadow to show that the element is hovered.